analysis/report_templates/default.html

```diff
@@ -100,6 +100,24 @@ <h2 class="green-text text-darken-3">experiment summary</h2>
 
             <h2 class="green-text text-darken-3">{{ benchmark.name }} summary</h2>
 
+            {% if benchmark.fuzzers_with_not_enough_samples %}
```

Comment:
Should this also check the in_progress attribute, since i think we only want to show this at end of experiment and not in real-time [since measurer might be behind].

Reply:
+1

Reply:
Good idea, done.